src/core/dev_api/openvino/core/type/element_iterator.hpp

```diff
@@ -576,5 +576,15 @@ template <Type_t ET, class T = typename std::add_const<ov::fundamental_type_for<
 constexpr auto iterator(const void* ptr) -> decltype(iterator<ET, T>(reinterpret_cast<T*>(ptr))) {
     return iterator<ET, T>(reinterpret_cast<T*>(ptr));
 }
+
+/**
+ * @brief Gets size of N elements of given precision in bytes.
+ *
+ * @param type  Element precision.
+ * @param n     Number of elements.
+ *
+ * @return Elements size in bytes.
+ */
+OPENVINO_API size_t get_byte_size(const element::Type& type, const size_t n);
```

Comment:
`get_byte_size` looks a bit confusing. How about `get_memory_size` with a description like `@return Memory size in bytes required to store n elements of the provided type`?